app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getMessage } from "../redux/messages/messageSlice";
+import { useEffect } from "react";
+
+
+function Greeting() {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMessage());
+  }, [dispatch]);
+
+  const message = useSelector((state) => state.messages.message);
+
+  return (<h1>
+    { message }
+  </h1>)
+}
+
+export default Greeting;
```

Comment:
- [ ] You have made this Greeting Component 👍 but I don't understand where you are using it 😨 Because you should have imported it into your App.js to make this component work 🤔 Kindly fix this problem to make your app work as expected 😊

Reply:
- [ ] Updated App.js to include the Greeting component. ✔ 
- [ ] Show random greeting messages.✔